docs/auditlogssource/README.md

```diff
@@ -0,0 +1,171 @@
+# Cloud AuditLogsSource Example
+
+## Overview
+
+This sample shows how to Configure a `AuditLogsSource` resource to 
+read data from [Cloud Audit Logs](https://cloud.google.com/logging/docs/audit/) and directly publish to the underlying transport (Pub/Sub), 
+in CloudEvents format.
+
+## Prerequisites
+
+1. [Install Knative with GCP](../install/README.md).
+
+1. [Create a Pub/Sub enabled Service Account](../install/pubsub-service-account.md)
+
+1. Enable the `Cloud Audit Logs API` on your project:
+
+   ```shell
+   gcloud services enable logging.googleapis.com 
+   gcloud services enable stackdriver.googleapis.com 
+   ```
+
+## Deployment
+
+1. Create a [auditlogssource](./auditlogssource.yaml).
+   This `AuditLogsSource` will get Cloud Audit Log Entries for topic creation.  
+   You can change the `serviceName`, `methodName` and `resourceName` to select the Cloud Audit Log Entries you want to view.
```

Comment:
maybe add a link to the public documentation where this fields are explained... https://cloud.google.com/logging/docs/reference/audit/auditlog/rest/Shared.Types/AuditLog